mteb/tasks/Reranking/fr/AlloprofReranking.py

```diff
@@ -10,13 +10,13 @@ class AlloprofReranking(AbsTaskReranking):
         name="AlloprofReranking",
         description="This dataset was provided by AlloProf, an organisation in Quebec, Canada offering resources and a help forum curated by a large number of teachers to students on all subjects taught from in primary and secondary school",
         reference="https://huggingface.co/datasets/antoinelb7/alloprof",
-        hf_hub_name="mteb/alloprof-reranking",
+        hf_hub_name="antoinelb7/alloprof",
```

Comment:
```suggestion
        hf_hub_name="lyon-nlp/mteb-fr-reranking-alloprof-s2p",
```
We created this reranking dataset from the original one.